source/content.js

```diff
@@ -40,7 +40,7 @@ import preserveWhitespaceOptionInNav from './features/preserve-whitespace-option
 import addMilestoneNavigation from './features/add-milestone-navigation';
 import addFilterCommentsByYou from './features/add-filter-comments-by-you';
 import addProjectNewLink from './features/add-project-new-link';
-import removeProjectsTab from './features/remove-projects-tab';
+import removeProjects from './features/remove-projects';
```

Comment:
If we're renaming it we need to add an option migration to `background.js` exactly like https://github.com/sindresorhus/refined-github/blob/4251e3c2d539b3936c3dda8d9cc395525171d13d/source/background.js#L17-L19